website/docs/about/contribution.md

```diff
@@ -1,34 +1,200 @@
 ---
 title: Contribution
+description: How to collaborate with the HashiCorp Design Systems (HDS) team to contribute back to the system.
 navigation:
   order: 104
 ---
 
 !!! Info
 
-At this time we are not accepting external contributions from non-HashiCorp employees. Many resources and links on this page are only available to HashiCorp employees.
+At this time **we are not accepting external contributions from non-HashiCorp employees**.<br/>
+Many resources and links on this page are only available to HashiCorp employees.
+
+!!!
+
+## From “Contribution” to “Collaboration” Model
+
+Introducing a contribution model for a design system may not be as easy at it seems. There are structural factors that can undermine a team’s effort to set up a contribution model (even more, one that actually works). Here are just some of them:
+
+- Contributions often cover only small low-level parts of the design system; large contributions are complex by nature, multi-disciplinary and time-consuming, and so they are beyond the contributors’ possibilities in terms of time/availability, knowledge or skills.
+- Contributions need to be kept to the same standards of quality as the other artifacts produced by the design system team: this means that contributors need to adhere to the same rigorous process that a system team uses to get work done. This can be extremely tedious, for someone who (most of the time) just needs to have something done to unblock their work.
```

Comment:
```suggestion
- Contributions need to be kept to the same standards of quality as the other artifacts produced by the design system team; this means that contributors need to adhere to the same rigorous process that a system team uses to get work done. This can be extremely tedious for someone who (most of the time) just needs to have something done to unblock their work.
```